admin/app.js

```diff
@@ -81,10 +84,38 @@ app.use(helmet.contentSecurityPolicy({
 var oneYearInSeconds = 31536000
 app.use(helmet.hsts({
   maxAge: oneYearInSeconds,
-  includeSubDomains: true,
-  preload: true
+  includeSubDomains: false,
+  preload: false
 }))
 
+// identify azure by specific environment variable
+function isAzure () {
```

Comment:
Seems like a helper function to me.

I would think this can be moved outside of app.js to keep it cleaner.

@jon-shipley @GuyHarwood opinions?

Reply:
will create azure specific file for this